### DISCUSSION TITLE
disable CS?

### DISCUSSION THREAD

**Discussion by reinierdevl** [Bodmer/TFT_eSPI#1543] [New processor/display support or capability request] [answered]
I'm really stuck with pins at the moment on my NodeMCU. For me a usefull extension would be to connect the CS of the display to ground (so it is always selected) to free up another PIN and be able to disable CS in the library, like RTS.

**Comment by Bodmer** [comment, marked as answer]
All you need to do is not  #define TFT_CS, just comment it out.
